index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>website concert</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/d6d5f8b156.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header>
+ <section class="homepage">
+  <nav class="mobile-navbar">
+      <div class="nav-logo">
+          <div id="hamburger">
+              <i class="fa fa-bars"></i>
+          </div>
+          <div class="logo">
+            <a href="index.html"></a>
+            <img class="img-logo" src="./images/riana.png" alt="Logo">
+          </div>
+      </div>
+      <div class="mobile-menu hide" id="mobile-menu-id">
+          <button id="cross">
+            <img src="./images/Icon.png" alt="">
+          </button>
+          <nav class="menu-list">
+              <ul>
+                  <li><a href="index.html">Home</a></li>
+                  <li><a href="about.html">About</a></li>
+                  <li><a href="#services-id">program</a></li>
+                  <li><a href="#team-id">sponsor</a></li>
+                  <li><a href="#team-id">news</a></li>
+              </ul>
+          </nav>
+      </div>
+  </nav>
+  <div class="nav-top">
+      <div class="desktop-nav-list" id="nav-top-list">
+          <ul>
+              <li><a href=""><i class="fab fa-facebook-f"></i></a></li>
+              <li><a href=""><i class="fab fa-twitter"></i></a></li>
+              <li><a href="">English</a></li>
+              <li><a href=""> My Page</a></li>
+          </ul>
+      </div>
+  </div>
+  <div class="desktop-navbar">
+      <div class="logo">
+          <img src="./images/riana.png" alt="logo-image">
+      </div>
+      <div class="desktop-nav-list">
+          <ul>
+          
+              <li><a href="about.html">About</a></li>
+              <li><a href="#team-id">program</a></li>
+              <li><a href="#team-id"> Sponsor</a></li>
+              <li><a href="#team-id"> News</a></li>
+          </ul>
+      </div>
+  </div>
+  </nav>
+  <div class="welcome">
+    <h3 class="text">"Hello how are you!"</h3>
+    <h1 class="text">Concert King's Jimin</h1>
+  </div>
+  <div class="welcome-description">
+    <p class="welcome-text">
+       Stage King' Jimin wows fans and netizens with his exhilarating dance,
+       vocals and visuals after BTS's Busan concert
+    </p>
+  </div>
```

Comment:
- [ ] As requested by the previous reviewer, your landing page still does not fully make use of the provided Margins and padding which makes it not look similar to the provided Behance design. Kindly ensure that you use the correct margins and padding so that this design matches the Behance design.

**This is how I am seeing this section:**

[Screencast from 26-10-2022 07:11:25 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/198078813-6082e7ef-81fc-44c6-934f-aec1abd4ee7e.webm)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/198078944-45bc6668-a118-43d8-983d-c2b88dcf4196.png)